units/find_outdated_rollups.go

```diff
@@ -107,6 +108,14 @@ func (j *findOutdatedRollupsJob) Run(ctx context.Context) {
 			for _, result := range results.Results {
 				if _, ok := j.seenIDs[result.ID]; !ok {
 					j.createFTDCRollupsJobs(ctx, factories[i:], result)
+
+					count += 1
+					if count >= 1000 {
```

Comment:
nit: can we make this a constant, just like `maxDurationsSize`?

Reply:
Done.